### ISSUE TITLE
Wrong totalCommission/totalSelfstakeRewards values

### ISSUE THREAD

**Issue by przemerr** [LiskHQ/lisk-service#1819]
### Actual behavior
https://betanet-service.lisk.com/api/v3/pos/validators?name=przemer
`"totalCommission":"540270372808","totalSelfStakeRewards":"0","earnedRewards":"540270372808"` with commission set to `90%`
### Expected behavior

with commission set to 90% the `totalComission` shouldn't be equal `earneadRewards` and `totalSelfStakeRewards` shouldn't be equal `0`

### Steps to reproduce
go to the provided endpoint, and check the numbers

### Which version(s) does this affect? (Environment, OS, etc...)

lisk-service@0.7-beta.4

**Comment by sameersubudhi** [comment]
Hello @przemerr,
I think, this issue should also be resolved with the fixes for https://github.com/LiskHQ/lisk-service/issues/1788.

**Comment by przemerr** [comment]
oh, sorry, i thought the fix was already included in the current version 😬

**Comment by sameersubudhi** [comment]
You're right, the fix was shipped in v0.7.0-beta.4. We'll re-analyse the issue and address it in this issue. Thank you for reporting 🙏

**Comment by przemerr** [comment]
I see that other delegates have correct values, possible reason might be that I've set the commission in the offset period where there was no block rewards.
